config.proto

```diff
@@ -134,12 +134,14 @@ enum GameModeId {
   MODE_FGC = 4;
   // Built-in Rivals of Aether mode.
   MODE_RIVALS_OF_AETHER = 5;
+  // Built-in Rivals of Aether 2 mode.
+  MODE_RIVALS_2 = 6;
```

Comment:
Please could you put this below `MODE_CUSTOM` and give it the value `9` instead. Otherwise it will result in incorrect behaviour for users of devices that are already using firmware based on a configurator build of HayBox when they update (i.e. their keyboard modes would turn into RoA2 mode, and custom modes would turn into keyboard modes).

Reply:
Updated commit message and put MODE_RIVALS_2 as the last GameModeId in config.proto